modules/atomic-widgets/module.php

```diff
@@ -155,4 +156,17 @@ private function enqueue_scripts() {
 			true
 		);
 	}
+
+	private function enqueue_styles() {
```

Comment:
```suggestion
	public function register_styles() {
```

The module should only register styles, and the element should declare a style dependency.